images/bridge2/ui/session.html

```diff
@@ -84,18 +84,32 @@
   const sessionStart = data.Session && new Date(data.Session.Start).getTime();
 
   const translations = {};
+  const assistants = {};
   for (const e of events) {
-    if (e.Type !== "translation") {
-      continue
-    }
-    const src = e.Data.SourceEvent
-    if (!translations.hasOwnProperty(src)) {
-      translations[src] = []
+    let src;
+    switch (e.Type) {
+    case "translation":
+      src = e.Data.SourceEvent
+      if (!translations.hasOwnProperty(src)) {
+        translations[src] = []
+      }
+      translations[src].push({
+        lang: e.Data.Translation.target_language,
+        text: e.Data.Translation.segments.map(s => s.text).join(),
+      });
+      break;
+    case "assistant":
```

Comment:
at some point we might give an assistant its own voice, would you still want to call this event "assistant"?

Reply:
"voice" as in text-to-speech, or what do you mean by that? I'm open to suggestions.

Reply:
yes, i'm imagining some kind of text-to-speech. i think we might want to either call this "assistant-text", "assistant-transcription", or maybe even just "transcription". what do you think?

Reply:
any updated thoughts here?